schemas/log-details.json

```diff
@@ -0,0 +1,19 @@
+{
+  "type": "object",
```

Comment:
We should also provide `title` and `description` to this schema, so it's clearer for other developers.

Reply:
Oh, and the Travis tests require some fixing.

Reply:
Done. Not sure I added the best description.